src/lib/esnext.arraybuffer.d.ts

```diff
@@ -0,0 +1,58 @@
+interface ArrayBuffer {
+	/**
+	 * Read-only. The maximum length that this ArrayBuffer can be resized to (in bytes).
+	 */
+	readonly maxByteLength: number;
+
+	/**
+	 * Read-only. Whether this ArrayBuffer can be resized or not.
+	 */
+	readonly resizable: boolean;
+
+	/**
+	 * Resizes the ArrayBuffer to the specified size (in bytes).
+	 * @param newLength The new length, in bytes, to resize the ArrayBuffer to.
+	 */
+	resize(newLength: number): undefined;
+}
+
+/**
+ * ArrayBuffer constructor options
+ */
+interface ArrayBufferOptions {
+	maxByteLength?: number;
+}
+
+interface ArrayBufferConstructor {
+	new(byteLength: number, options?: ArrayBufferOptions): ArrayBuffer;
+}
+
+interface SharedArrayBuffer {
+	/**
+	 * Read-only. Whether this SharedArrayBuffer can be grow or not.
```

Comment:
```suggestion
	 * Returns true if this SharedArrayBuffer can grow.
```

Although, `growable` is self-explanatory, and the current explanation is redundant with it. If we must have a jsdoc here, I'd prefer to see an explanation of context or what different properties a growable SharedArrayBuffer has, or a link to an explanation.